cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -283,5 +283,9 @@ type AuditPolicyConfiguration struct {
 	LogDir string `json:"logDir"`
 	// LogMaxAge is the number of days logs will be stored for. 0 indicates forever.
 	LogMaxAge *int32 `json:"logMaxAge,omitempty"`
+	// WebhookConfigPath is the local path to webhook policy.
+	WebhookConfigPath string `json:"webhookConfigPath"`
```

Comment:
This struct is starting to look like it wants to be two structs. One for the log backend configuration and one for the webhook configuration.